Microsoft.SystemForCrossDomainIdentityManagement/Protocol/ProtocolExtensions.cs

```diff
@@ -42,6 +53,36 @@ public static HttpMethod PatchMethod
             }
         }
 
+        public static bool IsResourceType(this object jobject, string scheme)
```

Comment:
This is an extension function, so it should rather be in a separate class that advertises that, called ObjectExtensions, rather than being mixed in a class with methods that are not extensions.